Client/src/components/landing.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+// import logo from './logo.svg';
+import {Link} from 'react-router-dom'
+import '../App.css';
+import logo from '../images/MYtineraryLogo.png';
+import arrow from '../images/circled-right-2.png';
+import home from '../images/homeIcon.png'; //yo doy el nombre de importacion para luego usarlo, en este caso home.
+
+
+class LandingPage extends Component {
+  render() {
+    return (
+    
+      
+      <div className="landing">
```

Comment:
I would recommend getting rid of the div wrappers and just adding classes to the child elements.